services/client/client.go

```diff
@@ -0,0 +1,30 @@
+package client
+
+import (
+	"academy/model"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+//ConsultExternalService will call the Jokes API
+func ConsultExternalService() []model.Joke {
+	url := "https://official-joke-api.appspot.com/random_ten"
+	request, error := http.NewRequest("GET", url, nil)
+	if error != nil {
+		fmt.Println(error)
+	}
+	response, _ := http.DefaultClient.Do(request)
```

Comment:
Don't forget to handle your errors instead of ignoring them with `_`

Reply:
Added code to handle error